todo/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
It is a good practice to write tests to cover you code and test endpoints. Would recommend to do that.